vertical-pod-autoscaler/pkg/recommender/input/metrics/test/main.go

```diff
@@ -0,0 +1,157 @@
+package main
```

Comment:
This is code that I keep for debugging, but I plan to drop it in case everything is fine.